proxies/ruby/app.rb

```diff
@@ -15,3 +16,52 @@
     capabilities: %w[EdgeDB LocalBucketing]
   }.to_json
 end
+
+data_store = {
+  clients: {},
+  commandResults: {}
+}
+
+post '/client' do
+  request.body.rewind
+  data = JSON.parse request.body.read
+
+  client_id, sdk_key, wait_for_initialization, options =
+    data.values_at('clientId', 'sdkKey', 'waitForInitialization', 'options')
+
+  if client_id.nil?
+    status 400
+    return { message: 'Invalid request: missing clientId' }.to_json
+  end
+
+  begin
+    options = DevCycle::DVCOptions.new(
+      config_cdn_uri: options.fetch('configCDNURI', 'https://config-cdn.devcycle.com'),
```

Comment:
Is this how the defaulting works?

Reply:
Yup. I needed to do that, otherwise we'd be passing in nils and that would cause the sdk to throw

Reply:
TIL